Cargo.toml

```diff
@@ -12,3 +12,10 @@ repository = "https://github.com/rust-lang/ar_archive_writer"
 
 [dependencies]
 object = { version = "0.32.0", default-features = false, features = ["std", "read"] }
+
+[features]
+__test = [ "object/write" ]
+
+[dev-dependencies]
+# Use a self-reference as a way to enable a dependency's feature but only when testing.
+ar_archive_writer = { path = ".", features = ["__test"] }
```

Comment:
You can use `object = { version = "0.32.0", default-features = false, features = ["write"] }` here instead afaik.

Reply:
Huh, that works, nice! I was following the workaround suggested in <https://github.com/rust-lang/cargo/issues/2911>, but I guess the new resolver has made that redundant?